airbyte-integrations/connectors/destination-postgres/encrypt

```diff
@@ -0,0 +1 @@
+6Yjx3VKrf2
```

Comment:
whats this diff ? 🤔

Reply:
a file created by some tests, I think. Let me remove it